config/wpcalypso.json

```diff
@@ -53,6 +53,7 @@
 		"gutenboarding/style-preview-verticals": false,
 		"gutenboarding/alpha-templates": true,
 		"gutenboarding/language-picker": false,
+		"gutenboarding/new-launch": true,
```

Comment:
Let's roll to `wpcalypso` when rolling to prod, otherwise it might get a bit confusing for HEs debugging the flow.

We can get sufficient testing done with horizon ourself.